client/src/rpc/client.rs

```diff
@@ -100,19 +102,22 @@ impl Transport for RuntimeTransport {
 
 #[cfg(not(target_env = "sgx"))]
 struct GrpcTransport {
-    grpc_client: EnclaveRpcClient,
+    grpc_client: EnclaveRPCClient,
+    runtime_id: RuntimeId,
     endpoint: String,
 }
 
 #[cfg(not(target_env = "sgx"))]
 impl Transport for GrpcTransport {
     fn write_message_impl(&self, _ctx: Context, data: Vec<u8>) -> BoxFuture<Vec<u8>> {
-        let mut req = CallEnclaveRequest::new();
-        req.set_payload(data);
-        req.set_endpoint(self.endpoint.clone());
```

Comment:
Yeah but it was passed in an awkward way via the "endpoint" (because initially there was no "runtime_id" argument, this got added later) as an URI, e.g. `key-manager://<runtime-id-hex>`. But now that we have the runtime_id argument I see no sense to keep using the more complicated URI scheme.